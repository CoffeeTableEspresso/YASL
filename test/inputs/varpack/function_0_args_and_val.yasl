
fn f() {
    return 1, 2
}

fn g() {
    return .a, .b, .c
}

fn h() {
    return 'first', 'second'
}

let a, let b, let c = false, f()

echo a
echo b
echo c

a, b, c = true, g()

echo a
echo b
echo c

a, b, c = undef, h()

echo a
echo b
echo c
