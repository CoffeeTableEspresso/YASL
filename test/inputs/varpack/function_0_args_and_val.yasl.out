false
1
2
true
a
b
undef
first
second
